.release-version

```diff
@@ -1 +1 @@
-9.9.1
```

Comment:
Probably doesn't need to be a major release following semver but im not too fussy so up to you.

Reply:
Sorry, before seeing your comment I have already gone for merge soon after you approved.

Reply:
No worries not a problem